src/common/i18n/locales/en-US.json

```diff
@@ -1154,14 +1154,20 @@
   "promote": {
     "title": "Promote",
     "sub-title": "Promote your post",
-    "duration": "Duration"
+    "duration": "Duration",
+    "learn-more":"Learn more about promotion in our",
+    "faq-link":"/faq#how-promotion-work",
+    "faq":"FAQ"
   },
   "boost": {
     "title": "Boost",
     "sub-title": "Boost your post",
     "amount": "Amount",
     "slider-hint": "Drag the slider to adjust the amount.",
-    "hint": "Actual value of vote you receive may vary"
+    "hint": "Actual value of vote you receive may vary",
+    "learn-more":"Learn more about boosting in our",
```

Comment:
@Adesojisouljay No need `in our` text because it is difficult to translation in different languages. `Learn more about boosting`  should be enough

Reply:
That's right, will update that